### PR TITLE
Check if qemu/ipmi/seabios installed before infrasim init.

### PR DESCRIPTION
**Pull request by JuneZhou19**
Before package installation, it will check if any package exists and will confirm whether to overwrite existing package. 
@chenge3 @turtle-fly @fub2 @XiaowenJiang @xiar

### CODE REVIEW

File: infrasim/package_install.py [hunk 1]
```diff
@@ -63,9 +63,36 @@ def install_bintray_packages(repo, package):
...
+    # Check if package installed
+    try:
+        qemu_installed = run_command(cmd)
+    except CommandRunFailed:
+        install_package = True
+    else:
+        while True:
+            ans = raw_input(package+" already exists. Overwrite it? (Y/n)")
+            if ans.lower() not in ('yes', 'no', 'y', 'n'):
+                print("Sorry, I didn't understand that.")
```
Comment (fub2) [inline infrasim/package_install.py]: prompt message is not clear enough. something like "type answer with either yes, no, y or n" sounds better

Comment (JuneZhou19) [inline infrasim/package_install.py]: Refined it.

**Comment by emcinfrasim** [comment]
hahaha

**Comment by turtle-fly** [comment]
👍 good job!

**Pull request merged**